### ISSUE TITLE
[Misión][Cuenca de Sholazar] El górloc enfadado

### ISSUE THREAD

**Issue by hugomiguel445** [RoDNet/Bugtracker-Infinium-4.3.4#2312]
• Nombre: El górloc enfadado

• Enlace: http://es.wowhead.com/quest=12578

• Descripción: al llegar al  Poblado Caminamoho debemos invocar a Moodle(http://es.wowhead.com/npc=28122) y entregar la misión.

• Problema: al llegar e invocarlo no se puede entregar la misión.

P.D: esta misión es una de las ultimas de la cadena de misiones de los corazones frenéticos.

**Comment by RaulioWoW** [comment]
no, aun no funciona , hoy lo probe por todos lados y no hay caso ;s

**Comment by ivanlm90** [comment]
Corroboro que no funciona la misión, pasa tal cual cuenta hugomiguel445
